cheroot/connections.py

```diff
@@ -16,6 +17,11 @@
 
 import six
 
+try:
+    import backports.socketpair  # noqa: F401  # lgtm [py/unused-import]
```

Comment:
this is not as simple to import through `_compat` since it adds the function directly to the `socket` module - [link](https://github.com/mhils/backports.socketpair/blob/f834f594cc89481995f7e532138789996c55b47b/backports/socketpair/__init__.py#L56) - so an `import as` statement doesn't quite work.

let me know how you'd like to manage this.